model/eventModel.js

```diff
@@ -2,28 +2,45 @@ const Mongoose = require('mongoose');
 
 const LocationSchema = require('../util/schemas/locationSchema');
 
+//validator function for checking if dates are in the future when creating an event
+const dateInFuture = (value) => {
+    return value > Date.now();
+}
+
 const eventSchema = Mongoose.Schema({
     companyID: {
         type: Mongoose.Types.ObjectId,
         required: true
     },
     name: {
         type: String,
-        required: true
+        required: true,
+        minlength: [1, 'Event name has to be at least 1 character long.']
     },
     location: {
         type: LocationSchema,
         required: true
     },
     startTime: {
         type: Date,
-        required: true
+        required: true,
+        validate: {
+            validator: dateInFuture,
+            message: 'Start time has to be in the future.'
+        }
     },
     endTime: {
         type: Date,
-        required: true
+        required: true,
+        validate: {
+            validator: dateInFuture,
+            message: 'End date has to be in the future.'
+        }
+    },
+    description: {
+        type: String,
+        maxlength: [1000, 'Description maximum length is 1000 characters.']
```

Comment:
Why only a 1000 characters?

Reply:
It could be raised to more but we gotta factor in the readability on the mobile app. For now i'll leave it at 1000 but we can always change it to more if needed.

Reply:
The more detailed description should be in the attached document if needed.